package.json

```diff
@@ -9,6 +9,7 @@
     "big.js": "^6.1.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^6.0.2",
```

Comment:
- [ ] Kindly deploy the project online and include the correct live deploy link in the readme.